src/storage/v3/shard_rsm.cpp

```diff
@@ -449,6 +271,65 @@ using SpecificEdgeProperties = std::tuple<msgs::VertexId, msgs::Gid, SpecificEdg
 using SpecificEdgePropertiesVector = std::vector<SpecificEdgeProperties>;
 using AllEdgePropertiesVector = std::vector<AllEdgeProperties>;
 
+template <typename ReturnType, typename EdgeProperties, typename EdgePropertyDataStructure, typename Functor>
+std::optional<ReturnType> GetEdgesWithProperties(const std::vector<memgraph::storage::v3::EdgeAccessor> &edges,
```

Comment:
Are these (this and the function below) function used anywhere?

Reply:
GetEdgesWithProperties, SetFinalEdgeProperties weren't used so I removed them both.